samples/apps/copilot-chat-app/webapp/src/components/chat/tabs/DocumentsTab.tsx

```diff
@@ -176,7 +147,8 @@ export const DocumentsTab: React.FC<ChatResourceListProps> = ({ chatId }) => {
                     accept=".txt,.pdf,.md,.jpg,.jpeg,.png,.tif,.tiff"
                     multiple={true}
                     onChange={() => {
-                        handleImport();
+                        // TODO: render error if image validation fails
+                        void fileHandler.handleImport(setDocumentImporting, documentFileRef, setSelectedDocuments);
```

Comment:
can we go ahead and throw the error here?

```suggestion
                        fileHandler.handleImport(setDocumentImporting, documentFileRef, setSelectedDocuments).catch((error) => {
                            const message = `Error validating image: ${String(error)}`;
                            dispatch(
                                addAlert({
                                    type: AlertType.Error,
                                    message,
                                }),
                            );
                        });
```

Reply:
meant to remove this - error handling is handled in useFile